### ISSUE TITLE
HTTP 500 – body: opening pr: non-fast-forward update:[...]

### ISSUE THREAD

**Issue by ahmetb** [rajatjindal/krew-release-bot#41]
I've just released v0.8.0 kubectx repo which has ctx and ns plugins.
One worked.
One failed.

https://github.com/ahmetb/kubectx/runs/459044199

Error:

```
time="2020-02-20T23:13:07Z" level=info msg="using template file \"/github/workspace/.krew/ns.yaml\""
time="2020-02-20T23:13:07Z" level=info msg="getting sha256 for https://github.com/ahmetb/kubectx/archive/v0.8.0.tar.gz"
time="2020-02-20T23:13:07Z" level=info msg="downloaded file /tmp/228467540/1582240387"
time="2020-02-20T23:13:09Z" level=fatal msg="expected status code 200 got 500. body: opening pr: non-fast-forward update: refs/heads/ahmetb-kubectx-v0.8.0\n"
```

Can you please investigate @rajatjindal? Thanks.

**Comment by rajatjindal** [comment]
Hi Ahmet

Thanks for reporting it. If i understand it right, i am surprised we didnt see this earlier.

I will check once i am online on laptop and fix it.

**Comment by ahmetb** [comment]
I am suspecting race condition, or not proper cleanup and/or reuse of container or concurrency>1. 😄

**Comment by rajatjindal** [comment]
Hi Ahmet,

I've released version v0.0.37 that potentially fixes this issue. I'll submit a PR for the ctx repo.
